tests/framework/web/AssetBundleTest.php

```diff
@@ -271,3 +287,20 @@ class TestAssetCircleB extends AssetBundle
         'yiiunit\\framework\\web\\TestAssetCircleA'
     ];
 }
+
+class TestAssetPerFileOptions extends AssetBundle
+{
+    public $basePath = '@testWebRoot';
+    public $baseUrl = '@testWeb';
+    public $css = [
+        'default_options.css',
+        ['tv.css', 'media' => 'tv'],
+        ['screen_and_print.css', 'media' => 'screen, print']
+    ];
+    public $js = [
+        'normal.js',
+        ['defered.js', 'defer' => 'defer'],
```

Comment:
@SilverFire @samdark 
How can I get 'defer' to be rendered as an attribute without a value ? 'defer' => null or 'defer' => false doesn't not work.
Is that supported in Yii?